index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;700&family=Open+Sans&family=Poppins:ital,wght@1,200&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--Header part start-->
+    <header>
+      <nav class="navbar">
+        <img src="images/Logo Placeholder.png" alt="logo" />
+        <a class="toggler" target="_self" href="nav-mobile.html">
+          <img src="images/Icon - Menu.png" alt="icon image" />
+        </a>
+      </nav>
+    </header>
+    <!--Main part start-->
+    <main>
+      <article class="main">
+        <h2>Hey There.
+            I’m Sazzad</h2>
+        <h3>I’am a Software Developer</h3>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+      </article>
+      <div class="social-media">
+        <ul class="social-media-icons">
+          <li>
+            <a href="https://github.com/SAZZAD1Q2" target="_blank" rel="noopener">
+                <img src="images/github.png" alt="">
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+                <img src="images/linkedin.png" alt="">
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+                <img src="images/finger.png" alt="">
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+                <img src="images/twitter.png" alt="">
+            </a>
+          </li>
+        
+          <li>
+            <a href="#" target="_blank">
+                <img src="images/M-img.png" alt="">
+            </a>
+          </li>
+        </ul>
+        <img class="main-img" src="images/main-img.png" alt="">
+      </div>
```

Comment:
You are importing png images for your `Logo name`, `nav-icon or burger-icon`, and here in social media icons, which is fine. But for various design sake, if you had to switch the color of these icons, then you would have to create a new icon with that image and import them in png. 
The solution for this is to use either `icons` instead of images (Libraries like font-awesome etc) or use `svg`s. Then you can change or switch the colors with the `color` property like the text elements property. 
What do you think about this suggestion?